src/electron/build_main.action.mjs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Outline Authors
```

Comment:
main is the node part, right? I was debating whether we should call this `electron/build_backend`

Reply:
(I split this out cuz it can then be separately cached)

Reply:
"backend" is too generic and under-specified. "Electron" main is well understood.